material-popup-menu/src/main/java/androidx/appcompat/widget/MaterialRecyclerViewPopupWindow.kt

```diff
@@ -89,6 +89,12 @@ class MaterialRecyclerViewPopupWindow(
 
     private val contextThemeWrapper: Context
 
+    private val windowManager: WindowManager = context.getSystemService(Context.WINDOW_SERVICE) as WindowManager
```

Comment:
Maybe initialize this lazily via `by lazy { }` since this is only used if dimming is enabled?

Reply:
Good idea. Done.